code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -131,6 +139,8 @@
 /datum/eldritch_knowledge/flesh_grasp/proc/remove_ghoul(datum/source)
 	var/mob/living/carbon/human/humie = source
 	spooky_scaries -= humie
+	humie.setMaxHealth(ORIGINAL_MAX_HEALTH)
```

Comment:
this should either be setting it to their initial max health, or use the pre-existing define `MAX_LIVING_HEALTH`